plugins/com.xored.q7.quality.mockups.swt/src/org/eclipse/form/FormToolkitTest.java

```diff
@@ -44,34 +47,40 @@ public Control construct(Composite parent) {
 		form.setFocus();
 		form.getBody().setLayout(new GridLayout());
 
-		Button button = toolkit.createButton(form.getBody(), "Test button",
-				SWT.NULL);
-		final Label label1 = toolkit.createLabel(form.getBody(), "Test label",
-				SWT.NONE);
-		// Boolean f = label1.getVisible();
-		// System.out.print(f);
+		view.getViewSite().getActionBars().getStatusLineManager().setMessage(null);
+		setClearStatusLineOnDispose(form);
+
+		Button button = toolkit.createButton(form.getBody(), "Test button", SWT.NULL);
+		final Label labelA = toolkit.createLabel(form.getBody(), "Label for mouse clicking test", SWT.NONE);
+		final Label labelB = toolkit.createLabel(form.getBody(), "Test label", SWT.NONE);
+
 		button.addSelectionListener(new SelectionAdapter() {
 			public void widgetSelected(SelectionEvent e) {
-				// TODO Auto-generated method stub
-				label1.setText("Pressing on the button");
+				labelB.setText("Button pressed. SelectionListener.widgetSelected called. StateMask: " + String.valueOf(e.stateMask));
+				labelB.setSize(500, 18);
 				// label1.setBounds(10, 25, 200, 25);
 			}
 		});
 
+		addMouseListener(button, "Button", button.getText());
+		addMouseListener(labelA, "Label", labelA.getText());
+
 		final Text tt = toolkit.createText(form.getBody(), "test");
 
-		Hyperlink link = toolkit.createHyperlink(form.getBody(), "Test link",
-				SWT.NONE);
+		Hyperlink link = toolkit.createHyperlink(form.getBody(), "Test link", SWT.NONE);

```

Comment:
Does this affect existing tests?

Reply:
No. On the build with new changes of the mockups, the same 10 of 132 mockup tests fails. The same as before.